Week1/MAKEME.md

```diff
@@ -7,18 +7,23 @@
 
 ### 1. Testing Exercises
 
+Using **Slack** application on desktop and mobile:
+
+- Describe shared testing activities for desktop and mobile versions
+
 ### 2. Assignment
 
-Given an application with web, desktop and mobile versions (e.g. WhatsApp):
+Given [RealWorld example application](https://github.com/gothinkster/realworld) on web ([React/Redux](https://github.com/gothinkster/react-redux-realworld-example-app)) and on mobile ([NativeScript (Angular)](https://github.com/nea/nativescript-realworld-example-app))
```

Comment:
For mobile it would be more compatible if it were React Native, but not sure if it's ready to be used already: [Real World App React Native](https://github.com/claytonschneider/realworld-react-native)

Reply:
I agree but this one is still in progress so it's not very stable. And I the point from mobile is just the testing aspect we wouldn't use it for automation later.